.github/workflows/PR-build.yml

```diff
@@ -36,8 +36,34 @@ permissions:
   contents: read
 
 jobs:
+  validate-markdown:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Get changed files
+        id: changes
+        run: |
+          echo "MDS=$(git diff --name-only --diff-filter=ACMRTUXB $(git merge-base origin/${{ github.event.pull_request.base.ref }} ${{ github.event.pull_request.head.sha }}) ${{ github.event.pull_request.head.sha }} | grep .md$ | xargs)" >> $GITHUB_ENV
+
+      - name: Install markdown-link-check
+        run: npm install -g markdown-link-check
+        if: ${{ env.MDS }}
+
+      - name: Run markdown-link-check
+        run: |
+          markdown-link-check \
+            --verbose \
+            --config .github/config/markdown-links-config.json \
+            ${{ env.MDS }} \
+            || { echo "Check that anchor links are lowercase"; exit 1; }
+        if: ${{ env.MDS }}
+  
   create-test-ref:
     runs-on: ubuntu-latest
+    needs: validate-markdown
```

Comment:
```suggestion
```
I don't think there needs to be a dependency between these two. Validating markdown can execute independently.